osu.Game/Overlays/Profile/Sections/ProfileItemBackground.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using System.Collections.Generic;
+
+namespace osu.Game.Overlays.Profile.Sections
+{
+    public class ProfileItemBackground : OsuHoverContainer
+    {
+        protected override IEnumerable<Drawable> EffectTargets => new[] { background };
+        protected override Container<Drawable> Content => content;
+
+        private readonly Box background;
+        private readonly Container content;
+
+        public ProfileItemBackground()
+        {
+            RelativeSizeAxes = Axes.Both;
+            Enabled.Value = true; //manually enabled, because we have no action
```

Comment:
Not sure how I feel about this from a UX standpoint. Yes, I understand it's to bring it with parity in respect to web, but it kind of breaks user expectations (it has a hover state and hover *and click* sounds, but clicking it doesn't actually do anything).

Reply:
Valid point. Removing sounds might work, but I'll wait for ppy's response on this.

Reply:
Actually, yes. In the current implementation there are no sounds, so I'll change that